packages/odyssey-react-mui/src/OdysseyTranslationProvider.tsx

```diff
@@ -47,6 +47,7 @@ export const OdysseyTranslationProvider = ({
   useEffect(() => {
     // Defaults to the browser's language if available otherwise `en` will be used
     i18n.changeLanguage(languageCode || window.navigator.language);
+    document?.documentElement?.setAttribute("lang", languageCode || "en");
```

Comment:
I think we can remove the `?` if we do `window.ownerDocument` or `window.document`.

Reply:
@KevinGhadyani-Okta I can change this to `window.document` for sure. We _may_ still need the optional chaining though in case this ever runs in some non-browser environment. Unless you don't see that as a potential issue

Reply:
Or some other explicit check that window is defined

Reply:
`useEffect` only runs in the browser, so we're good there!

Reply:
Wait, React v18 added server execution, but I don't know if it's `useEffect`.

Don't worry about it. We'll fix these issues if they come up. Everything at Okta is client-side, so I don't wanna be worrying about server-side execution.

Reply:
Right on. I was thinking that it might fail in CI trying to execute that line in a test or something